manager/src/test/java/com/redhat/service/bridge/manager/ProcessorServiceTest.java

```diff
@@ -134,26 +140,34 @@ public void createProcessor_bridgeDoesNotExist() {
 
     @Test
     public void createProcessor_processorWithSameNameAlreadyExists() {
-        Bridge b = createPersistBridge(BridgeStatus.READY);
+        Bridge b = createPersistBridge(ManagedResourceStatus.READY);
         ProcessorRequest r = new ProcessorRequest("My Processor", createKafkaAction());
 
         Processor processor = processorService.createProcessor(b.getId(), b.getCustomerId(), r);
-        assertThat(processor).isNotNull();
+        await().atMost(5, SECONDS).untilAsserted(() -> {
```

Comment:
A re-occurring theme in these tests...

`Processors` are now created asynchronously and hence may not be available immediately.

This caused some tests to intermittently fail as the test "completed" and the database cleaned up before the VertX worker had finished processing the work.. This lead to various manifestations of database updates being out of sync etc.